test/ragnarok/ManagedPortfolio.test.ts

```diff
@@ -320,8 +332,31 @@ describe('ManagedPortfolio', () => {
     })
   })
 
+  describe('isSignatureValid', () => {
+    it('returns true for valid signature', async () => {
+      const signature = await signMessage(lender, DEPOSIT_MESSAGE)
+      expect(await portfolio.isSignatureValid(lender.address, signature)).to.be.true
+    })
+
+    it('returns false for invalid message', async () => {
+      const signature = await signMessage(lender, 'other message')
+      expect(await portfolio.isSignatureValid(lender.address, signature)).to.be.false
+    })
+
+    it('returns false for invalid signer', async () => {
+      const signature = await signMessage(manager, DEPOSIT_MESSAGE)
+      expect(await portfolio.isSignatureValid(lender.address, signature)).to.be.false
```

Comment:
Can we also have a test where signature is signed by lender, but isSignatureValid() called for e.g. manager?

Reply:
`isSignatureValid` is a view function, that does not depend on `msg.sender` in any way